mindjunkies/lecture/views.py

```
@@ -181,6 +206,7 @@ def __init__(self, **kwargs: Any):
 
     def dispatch(self, request, *args, **kwargs):
         """Ensure the course exists before proceeding"""
+        
```

Ruff (0.8.2)

Blank line contains whitespace

Remove whitespace from blank line

(W293)